src/constants.ts

```diff
@@ -47,7 +47,18 @@ export namespace LanguageServerConstants {
   export const Latest = 'latest';
   export const LatestVersion = '3.5.0';
   export const UnknownVersion = 'unknown';
+  export const DafnyGitUrl = 'https://github.com/dafny-lang/dafny.git';
   export const DownloadBaseUri = 'https://github.com/dafny-lang/dafny/releases/download';
+  export const Z3VersionForCustomInstallation = '4.8.5';
+  export function GetZ3FileNameOSX(): string {
+    const z3v = LanguageServerConstants.Z3VersionForCustomInstallation;
+    return `z3-${z3v}-x64-osx-10.14.2`;
+  }
+  export function GetZ3DownloadUrlOSX(): string {
+    const z3v = LanguageServerConstants.Z3VersionForCustomInstallation;
+    const z3filenameOsx = GetZ3FileNameOSX();
+    return `https://github.com/Z3Prover/z3/releases/download/Z3-${z3v}/${z3filenameOsx}.zip`;
+  }
```

Comment:
nit: You may want to move these functions to the `dafnyInstallation.ts` module, similar to this function:
https://github.com/dafny-lang/ide-vscode/blob/1dcc0756e5b7b391b0091ed97a07c1821d7f1f75/src/language/dafnyInstallation.ts#L75-L80

As of now, I don't think we'll use these functions outside of the respective module anytime soon. But I'll leave it up to you 👍